docs/kpod.1.md

```diff
@@ -1,10 +1,10 @@
-% kpod(8) # kpod - Simple management tool for pods and images
+% kpod(1) kpod - Simple management tool for pods and images
 % Dan Walsh
-% SEPTEMBER 2016
-# NAME
+# kpod "1" "September 2016" "kpod"
+## NAME
 kpod
```

Comment:
No synopsis/description here?  Maybe:

kpod - Simple management tool for pods and images